src/theme/components/Layout.tsx

```diff
@@ -47,6 +48,26 @@ const Layout = ({ children, title }: Props): JSX.Element => (
       <GlobalStyle />
       <Helmet title={title} />
       {children}
+      <CookieConsent
+        location="bottom"
+        cookieName={'ga-disable-G-D2D0FR22RN'}
+        enableDeclineButton={true}
+        flipButtons={true}
+        cookieValue={false}
+        declineCookieValue={true}
+        onDecline={() => {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          // This function is installed in javascript by the plugin gatsby-plugin-google-gtag
+          gaOptout();
+
+          // Clean the unnecessary cookies
+          Cookies.remove('_ga', { path: '/' });
+          Cookies.remove('_ga_D2D0FR22RN', { path: '/' });
+        }}
+      >
+        This website uses cookies to enhance the user experience.
```

Comment:
⚠️ we may provide a more specific message to not scare users.
We are only using cookies for audience monitoring with google analytics. We could mention it in the message

Reply:
It's possible. I let you change the messsage, if you find that it is not enough explicit 😉

Reply:
Done in e384aa4

If we add a message more explicit, I'm afraid to scare the users.

Reply:
That's fine, thanks